tools/c7n_gcp/tests/test_gke.py

```diff
@@ -0,0 +1,88 @@
+# Copyright 2019 Capital One Services, LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from gcp_common import BaseTest, event_data
+
+
+class KubernetesClusterTest(BaseTest):
+
+    def test_cluster_query(self):
+        project_id = "cloud-custodian"
+
+        factory = self.replay_flight_data('gke-cluster-query', project_id)
+        p = self.load_policy(
+            {'name': 'all-gke-cluster',
+             'resource': 'gcp.gke-cluster'},
+            session_factory=factory
+        )
+        resources = p.run()
+        self.assertEqual(len(resources), 1)
```

Comment:
so we query massive amounts of data, but could return a {} and this test would still pass. please validate attributes on resources.

Reply:
Updated